src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -11676,6 +11687,17 @@ public abstract partial class MethodInfo : System.Reflection.MethodBase
         public static bool operator ==(System.Reflection.MethodInfo? left, System.Reflection.MethodInfo? right) { throw null; }
         public static bool operator !=(System.Reflection.MethodInfo? left, System.Reflection.MethodInfo? right) { throw null; }
     }
+    public sealed partial class MethodInvoker
+    {
+        internal MethodInvoker() { }
+        public object? Invoke(object? obj, System.Span<object?> arguments) { throw null; }
+        public object? Invoke(object? obj) { throw null; }
+        public object? Invoke(object? obj, object? arg1) { throw null; }
+        public object? Invoke(object? obj, object? arg1, object? arg2) { throw null; }
+        public object? Invoke(object? obj, object? arg1, object? arg2, object? arg3) { throw null; }
+        public object? Invoke(object? obj, object? arg1, object? arg2, object? arg3, object? arg4) { throw null; }
+        public static System.Reflection.MethodInvoker Create(System.Reflection.MethodInfo method) { throw null; }
```

Comment:
This should take `MethodBase` so that it can be used to invoke constructors on already-allocated types.

https://github.com/dotnet/runtime/issues/85539#issuecomment-1540709299

Reply:
Thanks missed that earlier comment.